.github/workflows/deploy-env.yml

```diff
@@ -0,0 +1,56 @@
+name: Deploy
+on:
+  push:
+    branches:
+      - master
+jobs:
+  deploy:
+    name: Deploy to Dev
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.13
+      - name: Install pnpm
+        run: npm install -g pnpm
+      - name: Install dependencies
+        run: ./scripts/install.sh
+      - name: Build all packages
+        run: ./scripts/build-all-packages.sh
+      - name: Deploy
+        env:
+          STAGE_NAME: e2etest
```

Comment:
Ah, just realized the environment with the stage name `e2etest` is already created and configured with all the assets, so changing it would mean creating a new env and reconfigure everything. I would leave it as is for now. Please let me know if you would still like me to change it.